### PR TITLE
Do not (unnecessarily) require preprocessing for fixed 32/64 bit regression tests

### PR DESCRIPTION
**Pull request by tautschnig**
This is not portable as some architectures might not provide header
files for non-native bit widths. Preprocessing/the use of header files
is not essential for any of these regression tests.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 9b809beeeb221d520f956e421fbfca523349d0b7).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/78355629

File: regression/cbmc/Visual_Studio_Types1/main.i [hunk 1]
```diff
@@ -1,7 +1,7 @@
 int main()
 {
   // these types are Visual Studio-specific
-  #ifdef _MSC_VER
+  /* #ifdef _MSC_VER */
```
Comment (smowton) [inline regression/cbmc/Visual_Studio_Types1/main.i]: ? Do you mean to expose other compilers to these?

Comment (tautschnig) [reply regression/cbmc/Visual_Studio_Types1/main.i]: We set `--i386-win32` in the command line for this test, so really we are simulating Visual Studio.

Comment (smowton) [reply regression/cbmc/Visual_Studio_Types1/main.i]: In that case I'd suggest remove the `ifdef`, I at least was pretty confused what you intended by a commented ifdef.

Comment (tautschnig) [reply regression/cbmc/Visual_Studio_Types1/main.i]: Done.

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 0a990efb9bf3e8853d0ca9ad8300482e54c6bc6e).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/78472192

**Review by smowton — APPROVED** [review]

**Review by kroening — APPROVED** [review]

**Review by peterschrammel — APPROVED** [review]

**Pull request merged**